docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-monitoring-stack.adoc

```diff
@@ -0,0 +1,82 @@
+[id="completing-prerequisites-for-migrating-ceph-monitoring-stack_{context}"]
+
+= Completing prerequisites for a {Ceph} cluster with monitoring stack components
+
+You must complete the following prerequisites before you migrate a {Ceph} cluster with monitoring stack components.
+
+.Procedure
+
+. Gather the current status of the monitoring stack. Verify that
+the hosts have no `monitoring` label (or `grafana`, `prometheus`, `alertmanager`
+in case of a per daemons placement evaluation) associated: 
+[NOTE]
+The entire relocation process is driven by cephadm and relies on labels to be
+assigned to the target nodes, where the daemons are scheduled. Review the https://access.redhat.com/articles/1548993[cardinality matrix] before assigning labels and choose carefully the nodes where the monitoring
+stack components should be scheduled on.
+//kgilliga: Should this "cardinality matrix" article be available only downstream?
```

Comment:
No, I mean, I don't see any equivalent upstream article so far, I would keep that link until we find something that can be described upstream, otherwise I have to rework the content of that KCS and have a dedicated section we can refer locally. wdyt? any particular thought on the approach we should follow?

Reply:
I think for now I will use an ifeval to make sure the link only renders downstream. Customers will only be able to access that link if they have customer portal credentials anyway. 

This is the closest doc I've found upstream, but I don't think it's what we want: https://docs.ceph.com/en/latest/releases/